RELEASE.md

```diff
@@ -0,0 +1,4 @@
+TBD
+===
+
+Work in progress.
```

Comment:
Do we need to add to this?

Reply:
V2 of the travis CLI used in our internal Github releaser will be able to directly use the content of that file as release note. But unfortunately for v1 (what we are using), we have to update the GitHub release note manually.

For now, we can either:
1. write the release note here and during release process, copy & paste to GitHub once a release is cut
2. or we just don't include this file for now and put the release note some where so that it can be copied & paste during release.

I think I prefer 1.

Reply:
Currently we add release notes to [CHANGELOG.md](https://github.com/aws/amazon-ecs-cli/blob/master/CHANGELOG.md) on each release. Since they'll continue being persisted here (unless you envision this not happening or changing) and we can c/p from here on release, does it make sense to omit this file until we can move to using travis CLI V2?

Reply:
Let's remove this file for now since we can continue to use `CHANGELOG.md` even for travis CLI v2.